### PR TITLE
OPS-1498: Merge cleave typography into shared styles

### PR DESCRIPTION
**Pull request by radicalray**

### CODE REVIEW

File: stylesheets/typography.scss [hunk 1]
```diff
@@ -117,11 +130,23 @@ strong {
...
-.text-nowrap {
+.bare-link {
+  &,
+  &:hover {
+    color: inherit;
+    text-decoration: none;
+  }
+}
+
+.nowrap {
```
Comment (radicalray) [inline stylesheets/typography.scss]: I will change this, don't worry!

File: stylesheets/typography.scss [hunk 2]
```diff
@@ -117,11 +130,23 @@ strong {
   font-weight: $weight-light;
 }
 
+.text-bold {
```
Comment (honigc) [inline stylesheets/typography.scss]: This class has already been added on line 112/125, plz remove

File: stylesheets/typography.scss [hunk 3]
```diff
@@ -26,8 +26,21 @@ $font-book: normal normal $weight-book $font-size-small $font-family-default, he
 $font-medium: normal normal $weight-medium $font-size-small $font-family-default, helvetica, sans-serif;
 $font-bold: normal normal $weight-bold $font-size-small $font-family-default, helvetica, sans-serif;
 
+.antialiasing {
+  -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```
Comment (honigc) [inline stylesheets/typography.scss]: Plz un-indent one level

File: stylesheets/typography.scss [hunk 4]
```diff
@@ -133,3 +158,7 @@ strong {
 .no-top-margin {
   margin-top: 0 !important;
 }
+
+.student-font {
+  font-family: "Castledown", sans-serif !important;
+}
```
Comment (honigc) [inline stylesheets/typography.scss]: Plz add a newline at the end of the file

File: stylesheets/typography.scss [hunk 5]
```diff
@@ -133,3 +158,7 @@ strong {
 .no-top-margin {
   margin-top: 0 !important;
 }
+
+.student-font {
```
Comment (honigc) [inline stylesheets/typography.scss]: We don't have this class in NDS, and I don't think we have Castledown available in NDS, so while I know we may want to eventually move towards making this part of our standards, there's some pre-work that needs to happen first. I think maybe this should be removed until then

**Pull request merged**